Makefile

```
@@ -0,0 +1,21 @@
+
+.BIN=./bin
+GONAME=cole
+VERSION := $(shell git describe --tags --always --dirty 2>/dev/null)
+
+all: build dockerbuild run
+
+build:
+	@echo "Building $(GOFILES) to ./bin"
+	@GOPATH=$(GOPATH) GOBIN=$(GOBIN) env GOOS=linux GOARCH=amd64 go build -ldflags "-X main.version=$(VERSION)" -o bin/ ./...
+dockerbuild:
```

If you do this

```suggestion
dockerbuild: build
```

it will always call build before running the docker build